nucypher/blockchain/eth/registry.py

```diff
@@ -364,7 +364,7 @@ def __repr__(self):
 
     @property
     def filepath(self) -> str:
-        return self.__filepath
+        return str(self.__filepath)
```

Comment:
Same comment here about `str(...)`: is this necessary? `self.__filepath` is only set to str values.

Reply:
Yeah... a little trouble with Pathlib introduction here.  Perhaps supl;superfluous.